Simong expedia contact help

gmsimon22 opened the pull request:
Here is my procedure on finding ways to contact Expedia support

caimile commented:
If you need help adding images, lmk and I can try to help!

## Introduction

- Love the marketing style.
- You could break up that paragraph by adding a break where it says "All you need to get started is..."
- Additionally, if you do end up breaking the paragraph there, I would phrase that to say "To get started, you'll need..."

## Instructions

- **Step 1:** Perhaps add the hyperlink here?
- **Steps 2 & 3:** You could add a breadcrumb trail here to condense these two steps into one. Something like "Navigate to **Study & Learn > Spaces > Reserve a Room**.
- **Steps 5 & 6:** I would clarify what it means to "select what kind of space/building you want." I know the implication here is that the user will have this screen up _while_ reading your instructions, but it might be helpful to acknowledge the placement of these features and to identify them as "filters."
- **Steps 7 & 8:** Double check this part of the procedure. I think if you perform Step 7, the page will refresh/update on its own, so you might want to switch the order of these two steps.
- **Step 9:** This one could use more clarification about how to preview what a room looks like. I would also mention how to identify which slots are open and which have already been taken (identified by whether the block is white or gray). Additionally, you could also mention here that blocks that are dark gray or black have been blocked out by the university itself for other purposes, whether it's because the library is closed then or because there is work being done to that room.
- **Step 12:** Turn the parenthetical info into a block quote. This could be an alert, since users are prohibited from reserving single slots longer than 2 hours.
- Is there a limit on the number of individual reservations a user can make? I thought at one point it was a maximum of 2 reservations? I could 100% be wrong, but this might be something to double check.

## Ideas for alerts

- You can reserve rooms up to a week in advance (except for some rooms, like 4502 Hunt Library)
- Some rooms are only accessible to graduate students, and they'll need their ID card to access
- You can add the reservation to your Google Calendar once you get the email from NCSU Libraries confirming your reservation

lily-soetebier reviewed (commented):
This procedure is in super good shape already, so I definitely have few comments than the last two, but here you go: 
## Staging
You do a really good job at hooking in and comforting what is likely a stressed and disoriented user. I think that separating out the pre-reqs in a different paragraph will help them stand out.

## Coaching
Your steps here are very clear and concise, and I can see how with the images not much else will need to be said.

## Alertiing
I think a good way to incorporate the alerting moves could be to add in notes about the most common mistakes such as:
* Needing to be able to log-in and likely dual authenticate NCSU credentials
* if there are any limits on number of bookings you can make at a time
* things that you should double check before submitting

Honestly this is in really really good shape, and with the images and tweaks to the alerting moves I think it will be good to go. Good work!

kb-boop commented:
## Staging

I love the tone of your paragraph--it's very warm and enthusiastic towards students who might be feeling new to campus. I wonder if it would help to add an unordered list of reasons you would use this page so that at a glance, students can determine if they should keep reading.

## Coaching

I'm curious as to why you didn't use a numbered list for the steps. Do you think the steps need to be numbered? Or does this distract from the instructions since there are so many steps.

On that same note, could you consider splitting up "instructions" steps into smaller subheadings? It seems like a lot of consecutive steps and the reader's eyes might get tired.

## Alerting

I'm not sure if there's any altering needed. Are there any things students should look out for when booking a library room?

gmsimon22 committed:
added outline

added outline and main headings for contact help branch

gmsimon22 committed:
Added content, images, and procedure

I added the content and images for the contact help procedure. I also put the images into different folders for organization.

gmsimon22 committed:
Changed image order

I changed the order of when the images appear to make it match with the other procedures and provide and better layout

gmsimon22 committed:
added html to adjust image sizes

I added html to adjust the image sizes

gmsimon22 committed:
changed the image sizes and put them in the right spots

I moved the html image text to their proper places in the procedures

WBlan97 commented on simong/p-expedia-contact-help.md:
```diff
@@ -0,0 +1,53 @@
+
+## Prerequisites
+You will need access to a computer and a stable WiFi network.
+
```
I do not think you need prerequisite for this task.

alyssajohnsonunofficial replied:
I agree with Wyatt here that a prerequisite most likely isn't needed for this task! I also want to note that I really like your introduction here. Once again very concise and helpful, and tells users exactly what they're getting from this and why you made the user guide. Also, I really love the diversity in the photos and screenshots you're including.

WBlan97 commented on simong/p-expedia-contact-help.md:
```diff
@@ -0,0 +1,53 @@
+
+<img src="/simong/images/images-help/how-can-we-help-textbox.jpeg" alt="How Can We Help Search Bar" style="width: 300px">
+*In the search bar, you can ask your inquiry to find a variety of resources*
+
```
Great images and clear directions here!

alyssajohnsonunofficial replied:
I agree with Wyatt, the directions here are very clear and you're using strong imperative verbs to start them off! I also think the images are implemented throughout the procedure in a way that feels well-organized and natural!

WBlan97 commented on simong/p-expedia-contact-help.md:
```diff
@@ -0,0 +1,53 @@
+<img src="/simong/images/images-help/virtual-agent.jpeg" alt="virtual agent form" style="width: 300px">
+
+Answering the prompted questions will guide the virtual assistant to put you in contact with the right customer representative 
+
```
I like the use of images here! Hopefully the customer representative will be able to help.

alyssajohnsonunofficial commented on simong/p-expedia-contact-help.md:
```diff
@@ -0,0 +1,53 @@
+
+<img src="/simong/images/images-help/contact-us-questions.jpeg" alt="contact form" style="width: 300px">
+
+## Reaching the Virtual Assistant
```
I really like that you've included this alternative! I think you do a good job explaining how to utilize this feature without turning this into a function-oriented procedure. More use of great imperative verbs!

alyssajohnsonunofficial reviewed (commented):
This one's looking really good so far! I think you've got a strong handle on user-friendly language and tone, and I like the alternative you included for the procedure!

gmsimon22 committed:
Fixed italics error

I fixed the unitalicized parts in Select Seats

gmsimon22 committed:
added spaces to line up the images with the text

I added spaces to line up the images with the text in the contact help procedure

gmsimon22 committed:
hid prerequisite content and changed images back

I hid prerequisite content and changed images back from the indentation

gmsimon22 committed:
fixed italics

I fixed some italics that got unitalicized

gmsimon22 committed:
Updated headings, about section, image captions

I changed the headings to fit the content better, added an about section for staging, changed the width of some images, and added captions to the images

gmsimon22 committed:
Added content to readme.md and user-scenario.md and added task analysis pdf

I added content to the readme.md and user-scenario.md . I also added the task analysis pdf to docs/assests

gmsimon22 committed:
added PDF and google doc links to user scenario

I added the PDF and google doc versions of my task analysis to my user-scenario.md

lingeringcode committed:
Feedback from Lindgren
